app/src/androidTest/java/com/sdp13epfl2021/projmag/database/UserProfileDatabaseTest.kt

```diff
@@ -12,7 +12,7 @@ import com.google.firebase.firestore.DocumentReference
 import com.google.firebase.firestore.DocumentSnapshot
 import com.google.firebase.firestore.FirebaseFirestore
 import com.sdp13epfl2021.projmag.JavaToKotlinHelperAndroidTest
-import com.sdp13epfl2021.projmag.database.impl.firebase.UserProfileDatabase
+import com.sdp13epfl2021.projmag.database.impl.firebase.FirebaseUserdataDatabase
```

Comment:
After the merge of userdata and profile, tests should also be merge inside a single file.